docs/glossary.md

```diff
@@ -0,0 +1,194 @@
+
+### KZG Commitments
+KZG commitments, introduced by Aniket Kate, Gregory M. Zaverucha, and Ian Goldberg in 2010, provide a way to commit to polynomials in a succinct manner. Recently, polynomial commitments came to the forefront, being primarily used as commitments in PLONK-like zero knowledge constructions.
+
+In Avail’s construction, we use KZG commitments for the following reasons:
+
+* It allows us to commit to values in a succinct manner to be kept inside the block header.
+* Short openings are possible which helps a light client verify availability.
+* The cryptographic binding property helps us avoid fraud proofs by making it computationally infeasible to produce incorrect commitments.
+
+
+
+### Erasure Coding
+
+It is a method of reproducing data from smaller encoded blocks. The light client recovers application data from the already erasure-encoded blocks on Avail. It’s important because the LC is sampling randomly, so block-level redundancy makes it more likely that the LC would find missing data.
+
+
+### Fraud Proofs
+Fraud proofs are a type of cryptographic proof that can be used to prove that a transaction or state transition is invalid. Fraud proofs are particularly useful for blockchain applications, where it is important to be able to verify the validity of transactions and state transitions quickly and efficiently.
+
+### Data Availability Committee (DAC)
+A Data Availability Committee (DAC) is a trusted group of data managers (nodes) that store and grant access to data, maintaining the data off the main chain. Upon receiving and verifying data requests, the DAC provides authorized users with the needed data.
+
+### Data Attestation
+Data attestation is the process of verifying the authenticity and integrity of data. A block header on Avail contains two types of attestations. One attestation being KZG polynomial commitments of the submitted data and another being the root of the Merkle tree whose leaves are the data blobs. Supermajority of validators of Avail reach finality over the header by signing over a chain containing the header using the GRANDPA protocol.
+
+
+
+## Account and Asset Management
+
+### Avail JS Apps
+The Avail JS Apps UI is a forked version of [Polkadot JS Apps UI](https://polkadot.js.org/apps/) repository for visualizing and interacting with the Avail network.
+### Stash
+It is a private and secure storage for accounts. It is typically implemented as a separate wallet or account that is not connected to the main account or wallet. Stashes can be used to store a variety of assets, including cryptocurrencies, NFTs, and other digital assets.
+
+### Controller
+Controller is a person or entity that has the authority to manage an account. Controllers can perform a variety of tasks, such as: Transferring assets, Receiving assets, Setting permissions, Changing account settings, and Viewing account information.
+
+### AVL
+The native token of Avail.
```

Comment:
continue the statement to mention what it's used for - there should be more clarity around this now with the incentivized testnet coming up